src/main/java/persistence/sql/dml/builder/DeleteQueryBuilder.java

```diff
@@ -0,0 +1,28 @@
+package persistence.sql.dml.builder;
+
+import persistence.meta.MetaEntity;
+
+import java.lang.reflect.Field;
+
+public class DeleteQueryBuilder<T> {
+  private static final String DELETE_SQL_QUERY = "DELETE FROM %s WHERE %s = %s;";
+
+  public String createDeleteQuery(T entity, Field field) {
+    MetaEntity<T> metaEntity = MetaEntity.of(entity);
+    Object value = getFieldValue(field, entity);
+    String dbFieldName = metaEntity.getFieldMapping().get(field.getName());
+    return String.format(DELETE_SQL_QUERY, metaEntity.getTableName(), dbFieldName, value);
+  }
+
+  private String getFieldValue(Field field, T entity) {
```

Comment:
```suggestion
  private String getFieldValue(Field field, Object entity) {
```

제네릭 타입을 사용하시는 이유가 별도로 있는지 궁금해요!

Reply:
Object 데이터 타입의 사용과 타입 캐스팅을 지양하기 위해서 Generic을 사용했습니다.

특정 T클래스(Entity)의 querybBuilder를 만들지 않을까? 라는 의도를 담았습니다.

혹시 Object로 변경되야하는 이유는 무엇일지 궁금합니다!